templates/certified/components/_hardware-details.html

```diff
@@ -0,0 +1,63 @@
+<section class="p-section">
+  <div class="row--25-75">
+    <div class="col">
+      <h2 class="p-muted-heading">Component details</h2>
+    </div>
+    <div class="col">
+      <table aria-label="Component details table">
+        <tbody>
+          <tr>
```

Comment:
Not sure if there's ever an instance where the model name would be missing, but we should add a check for the vendor name, `/component/688` shows an empty td here 
![image](https://github.com/user-attachments/assets/471f3ca5-c649-4102-9937-d554cd5b102c)